package.json

```diff
@@ -47,21 +47,22 @@
     "dist"
   ],
   "dependencies": {
-    "@microsoft/signalr": "^8.0.0",
+    "@microsoft/signalr": "^8.0.7",
     "axios": "^1.4.0",
     "ts-log": "^2.2.5"
   },
   "devDependencies": {
-    "@types/node": "^20.14.7",
-    "@typescript-eslint/eslint-plugin": "^5.59.5",
-    "@typescript-eslint/parser": "^5.59.5",
-    "eslint": "^8.40.0",
-    "eslint-plugin-simple-import-sort": "^10.0.0",
-    "eslint-plugin-tsdoc": "^0.2.17",
-    "prettier": "^3.3.2",
-    "ts-node": "^10.9.1",
-    "tsup": "^8.0.2",
-    "typescript": "^5.0.4",
-    "vitepress": "^1.1.3"
+    "@types/node": "^22.7.0",
```

Comment:
CI is still on node v20, so we should probably align everything to 22 if we're doing this

Reply:
Oh yeah forgot this was on node 20, will revert back to stay on lts no need to move